.spelling

```diff
@@ -0,0 +1,7 @@
+e.g.
```

Comment:
If we use several makefile targets for diferent languages, we may switch to .spelling-en and .spelling-de.

Reply:
This seems to not be needed. I didn't test but there seems to be an option to have a .spelling file relative to the path.

```
 ./node_modules/.bin/mdspell --help
...
  -t, --target-relative    Uses ".spelling" files relative to the target.
  -h, --help               output usage information
```